pkg/logging/options.go

```diff
@@ -69,6 +70,12 @@ func WithFilter(f FilterLogging) Option {
 	}
 }
 
+func WithLevelledLogger(f LevelledLogging) Option {
```

Comment:
I'm wondering what is actually the reason to have `WithLeveledLogger` and `WithFilter` to be functional options and exported? It's not like we support users to provide their own leveled logger or filter, or do we?

Reply:
I agree, I am not sure that the functional options are required here. However I am not sure if there were other plans for the `logging` package to be used elsewhere or what the initial driver for that code was. It does seem excessive for the way the package is currently used.

To answer your question in regards to this `LevelledLogger` - the input in the config determines the level to log at, which is why I am using this currently. I believe it could all be done in the constructor as I had commented previously